.quickstart/quickstart.yml

```diff
@@ -36,6 +36,13 @@ table_variables:
     - fulfillment_tracking_info
   shopify_gql_using_product_variant_media:
     - product_variant_media
+  shopify_gql__using_order_custom_attribute:
```

Comment:
I noticed the initial variables have single `_` between `gql` and `using` and these ones have double. Should we keep this consistent and keep it single `_`?